docs/develop/howtos/i18n.md

```diff
@@ -9,12 +9,35 @@ In order to translate strings, they have to be marked. This ensures that they wi
 ### Python
 
 !!! warning
-Do not use `f-strings` for translatable text. Interpolation is not supported for these types of strings (see [Issue](https://github.com/python-babel/babel/issues/594)).
+Do not use f‑strings for translatable text.
+F‑strings perform interpolation immediately, so the extracted message will contain the interpolated text (e.g. "Hello, max") rather than the intended placeholder (e.g. "Hello, {username}"). This means that your translators will see the wrong string and translations will not work as expected.
```

Comment:
i think that even with `f-strings`, the extracted message will still have the shape `"Hello, {username}"` because the extraction runs on static code by parsing the syntax tree, and doesn't need to run the code (which would be required to know the content of variables (unless they're hard-coded values)